src/zope/formlib/orderedSelectionList.pt

```diff
@@ -114,17 +114,9 @@ function copyDataForSubmit(name)
     {
     // create virtual node with suitable attributes
     var newNode = document.createElement("input");
-    var newAttr = document.createAttribute("name");
-    newAttr.nodeValue = name;
-    newNode.setAttributeNode(newAttr);
-
-    newAttr = document.createAttribute("type");
-    newAttr.nodeValue = "hidden";
-    newNode.setAttributeNode(newAttr);
-
-    newAttr = document.createAttribute("value");
-    newAttr.nodeValue = toSel.options[i].value;
-    newNode.setAttributeNode(newAttr);
+    newNode.setAttribute("name", name.replace(/-/g, '.')+':list');
+    newNode.setAttribute("type", "hidden");
+    newNode.setAttribute("value",toSel.options[i].value );
```

Comment:
Add a space after the comma, and remove space before `)`.

Reply:
Done.